run_all_tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-sbt clean scalafmtAll scalastyleAll compile coverage test it:test coverageOff coverageReport dependencyUpdates
+sbt clean scalafmtAll scalastyleAll compile coverage Test/test coverageOff coverageReport dependencyUpdates
```

Comment:
you are not running integration tests this way, please replace `Test/test` with `test IntegrationTest/test`